### PR TITLE
Added utf-8 char status icons for mpd widget

### PR DESCRIPTION
**Pull request by s-hell**
Added a new function to display UTF8 status icons in mpd widget.

### CODE REVIEW

**Comment by s-hell** [comment]
Ok, looks like some encoding errors. I played around with libqtile/pangocffi.py but i always get an UnicodeDecodeError.

Can anyone help me with this?!?

**Comment by flacjacket** [comment]
Which version of Python are you using? Getting unicode to play nicely between all Python versions can be a bit of a pain, but I know e.g. the Volume widget was able to do it.

What's the full traceback?

File: docs/manual/hacking.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ install them from your package manager.
 
 * `Nose <http://nose.readthedocs.org/en/latest/>`_
 * `Xephyr <http://www.freedesktop.org/wiki/Software/Xephyr>`_
+* `xrandr <www.x.org/wiki/Projects/XRandR/>`_
 * ``xeyes`` and ``xclock``
 
 On ubuntu, this can be done with ``sudo apt-get install python-nose
```
Comment (mjkillough) [inline docs/manual/hacking.rst]: Is `xrandr` included in this `apt-get` command? (I couldn't tell if it was included in `x11-apps` -  I am not familiar with Ubuntu).

**Comment by s-hell** [comment]
Different computer, now everything works fine. :-(

File: docs/manual/hacking.rst (hunk 1 above)
Comment (s-hell) [inline docs/manual/hacking.rst]: Checked it on a debian System. xrandr is part of x11-xserver-utils

**Comment by flacjacket** [comment]
For this to pass the 3.2 tests, you can't use `u""` strings. You can do as [volume.py](https://github.com/qtile/qtile/blob/develop/libqtile/widget/volume.py#L149) did to display Unicode by using `six.u`. The characters you need are `u("\u25b6")`, `u("\u25b7")`, and `u("\u25a0")`.

**Comment by s-hell** [comment]
Ok, that works for python3.2 but still breaks 2.7:

2015-12-08 08:23:54,945 qtile _get_status:284  Mpd error on update
Traceback (most recent call last):
  File "/home/shellmann/git/github/qtile/bin/libqtile/widget/mpdwidget.py", line 275, in _get_status
    pangocffi.markup_escape_text(text[:progress]),
  File "/home/shellmann/git/github/qtile/bin/libqtile/pangocffi.py", line 189, in markup_escape_text
    ret = gobject.g_markup_escape_text(text.encode(), -1)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u25b6' in position 0: ordinal not in range(128)

**Comment by flacjacket** [comment]
Hmm, maybe try this rebasing your commits on top of this:

https://github.com/flacjacket/qtile/tree/pango_utf8

This is the horrible murky area of getting Python 2 and 3 to agree on Unicode handling

**Comment by flacjacket** [comment]
I pulled that commit that I referenced in my previous comment, so you can check if this works if you rebase against the current develop branch.

**Comment by zordsdavini** [comment]
It should be abandoned as Qtile drops suport for <3.6 python version support

**Comment by tych0** [comment]
Yes, thanks, I'll close this for now.

**Pull request closed without merging**